gemini/prompts/prompt_optimizer/vertex_ai_prompt_optimizer_sdk.ipynb

```diff
@@ -361,7 +361,9 @@
         "-   `Storage Object Admin` to read and write to your GCS bucket.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    custom_job.run()
</pre><p>Please pass the service_account to this function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>